Tchap/Modules/Room/RoomViewController.m

```diff
@@ -2581,34 +2581,34 @@ - (void)showAdditionalActionsMenuForEvent:(MXEvent*)selectedEvent inCell:(id<MXK
         }
 #endif
         
-        NSString *titleKey;
-        if (![self.roomDataSource.room.accountData getTaggedEventInfo:selectedEvent.eventId withTag:kMXRoomTagFavourite])
+        if (!selectedEvent.isState)
         {
-            titleKey = @"room_event_action_favourite";
-        }
-        else
-        {
-            titleKey = @"room_event_action_remove_favourite";
-        }
-        [currentAlert addAction:[UIAlertAction actionWithTitle: NSLocalizedStringFromTable(titleKey, @"Vector", nil)
-          style:UIAlertActionStyleDefault
-        handler:^(UIAlertAction * action) {
-            
-            if (weakSelf)
+            NSString *titleKey;
+            if (![self.roomDataSource.room.accountData getTaggedEventInfo:selectedEvent.eventId withTag:kMXRoomTagFavourite])
             {
-                typeof(self) self = weakSelf;
+                titleKey = @"room_event_action_favourite";
+            }
+            else
+            {
+                titleKey = @"room_event_action_remove_favourite";
+            }
+            [currentAlert addAction:[UIAlertAction actionWithTitle: NSLocalizedStringFromTable(titleKey, @"Vector", nil)
+              style:UIAlertActionStyleDefault
+            handler:^(UIAlertAction * action) {
                 
+                MXWeakify(self);
```

Comment:
here you have to use "MXStrongifyAndReturnIfNil(self)"  instead of MXWeakify
Tell me if you don't see why